muxer.go

```diff
@@ -0,0 +1,411 @@
+package astits
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"github.com/asticode/go-astikit"
+	"io"
+)
+
+const (
+	StartPID           uint16 = 0x0100
+	PMTStartPID        uint16 = 0x1000
+	ProgramNumberStart uint16 = 1
+)
+
+var (
+	MuxerErrorPIDNotFound      = errors.New("PID not found")
```

Comment:
Replace the prefix `MuxerError` with `Err` and in the error message add `astits: ` at the beginning (result would be `ErrPIDNotFound = errors.New("astits: PID not found")`)

Reply:
Will change, though sometimes I get errors from astits like (it's just an example from my mind, not a real error message) "astits: error reading chunk: astits: can't read byte: astits: EOF", which is not very pretty